benchmarks/reduce.py

```diff
@@ -5,6 +5,7 @@
 import arkouda as ak
 
 OPS = ('sum', 'prod', 'min', 'max')
```

Comment:
suggestion: other reductions and types are supported but we might not really need them for benchmarking maybe just for testing correctness.
`int64`, `float64`, `bool` also support `any` and `all` reductions
`argmin`, `argmax`, and `is_sorted` are also supported for `int64` and `float64`

Reply:
Interesting, that's up to you -- If they have performance characteristics it might be worth adding them here, but if performance won't be much different, I'd probably leave them out of here and just rely on the other correctness tests for these ops